frontend/src/pages/login/loginPage.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+import {
+  IonButton,
+  IonCard,
+  IonCardContent,
+  IonContent,
+  IonHeader,
+  IonInput,
+  IonItem,
+  IonLabel,
+  IonNav,
+  IonPage,
+  IonSpinner,
+  IonTitle,
+  IonToolbar,
+  useIonToast,
+} from '@ionic/react'
+
+import './login.css'
+import Amplify from '@aws-amplify/core'
+import AuthState from '../../model/authState'
+import { useRegisterUpdate } from '../../model/registerContext'
+
+interface LoginPageProps {
+  setAuthState(authState: AuthState): void
+}
+
+const LoginPage: React.FC<LoginPageProps> = ({ setAuthState }) => {
+  const [mail, setMail] = useState('')
+  const [password, setPassword] = useState('')
```

Comment:
```suggestion
  const [userInfo, setUserInfo] = useState({email:'',password:''})
```
You could use an `object` instead of two separate useState and then set it with `setUserInfo(prevState => {...prevState, email:"123"}))`

Reply:
I don't see much value when it only consists of two properties. For more I see where it makes sense but if it's only 2....?